src/components/Footer.tsx

```diff
@@ -0,0 +1,55 @@
+import { FilterType } from '../types/FilterType';
+import { Todo } from '../types/Todo';
+import cn from 'classnames';
+
+type Props = {
+  filter: FilterType;
+  notCompletedTodosCount: number;
+  handleFilterChange: (newFilter: FilterType) => void;
+  filteredTodos: Todo[];
+  handleClearCompleted: () => Promise<void>;
+};
+
+export const Footer: React.FC<Props> = props => {
+  const {
+    notCompletedTodosCount,
+    filter,
+    handleFilterChange,
+    filteredTodos,
+    handleClearCompleted,
+  } = props;
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {notCompletedTodosCount} items left
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        {Object.values(FilterType).map(filterOption => (
+          <a
+            key={filterOption}
+            href={`#/${filterOption === FilterType.All ? '' : filterOption.toLowerCase()}`}
```

Comment:
the same with `&&` operator
```suggestion
            href={`#/${filterOption !== FilterType.All && filterOption.toLowerCase()}`}
```